pkg/controllers/nodeclaim/termination/controller.go

```diff
@@ -110,6 +122,13 @@ func (c *Controller) finalize(ctx context.Context, nodeClaim *v1beta1.NodeClaim)
 			return reconcile.Result{}, client.IgnoreNotFound(fmt.Errorf("removing termination finalizer, %w", err))
 		}
 		log.FromContext(ctx).Info("deleted nodeclaim")
+		InstanceTerminationDuration.With(map[string]string{
```

Comment:
Should this be right after we get past the instance termination rather than after we remove the finalizer? Since this metric is meant to just track the instance termination

Reply:
Agreed